.travis.yml

```diff
@@ -1,22 +1,33 @@
 language: python
 
 before_cache:
-- rm -f $HOME/.cache/pip/log/debug.log # erase log
+  - rm -f $HOME/.cache/pip/log/debug.log # erase log
```

Comment:
Is this still needed when using Travis's built-in definitions for the cache?

Reply:
Honestly I'm not sure. At one point caching was only available in the pro one unless you configured it yourself! I don't know if it dealt with such subtleties too, so I'm tempted to leave it given it doesn't really do any harm.